data/126/021/452/3/1260214523.geojson

```diff
@@ -6,21 +6,21 @@
     "edtf:inception":"uuuu",
     "geom:area":0.0,
     "geom:area_square_m":0.0,
-    "geom:bbox":"-10.2,11.73333,-10.2,11.73333",
+    "geom:bbox":"-10.18,11.73333,-10.18,11.73333",
```

Comment:
NOTE: These `geom` properties are auto-generated. To change them you'd need to edit the `geometry` section below (which it looks like you did, good). Generally don't manually edit these...